numpy/core/_add_newdocs.py

```diff
@@ -320,43 +320,42 @@
     Here is how we might write an ``iter_add`` function, using the
     Python iterator protocol::
 
-        def iter_add_py(x, y, out=None):
-            addop = np.add
-            it = np.nditer([x, y, out], [],
-                        [['readonly'], ['readonly'], ['writeonly','allocate']])
-            with it:
-                for (a, b, c) in it:
-                    addop(a, b, out=c)
-            return it.operands[2]
+        >>> def iter_add_py(x, y, out=None):
```

Comment:
I'm not sure these added indentation markers are an improvement in readability... it would be great if doctests could execute multiline function definitions with it.

Reply:
Would be nice -- maybe if / when we generate an ecosystem "scientific docstrings" package of sorts.